openstack_controller/datadog_checks/openstack_controller/components/component.py

```diff
@@ -80,7 +80,7 @@ def wrapper(self, *args, **kwargs):
                     if report_service_check:
                         self.check.service_check(self.SERVICE_CHECK, AgentCheck.CRITICAL, tags=tags)
                 except Exception as e:
-                    self.check.log.error("Exception: %s", e)
+                    self.check.log.debug("Exception [%s]: %s", type(e), e)
```

Comment:
```suggestion
                    self.check.log.debug("Encountered an exception in %s [%s]: %s", str(func), type(e), e)
```
what do you think about something like this?